packages/dd-trace/test/lambda/fixtures/handler.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+const _tracer = require('../../../../dd-trace')
+
+exports.handler = async (...args) => {
+  const sleep = ms => new Promise(resolve => setTimeout(resolve, ms))
+
+  await _tracer.trace('self.sleepy', async () => {
+    await sleep(200)
+  })
```

Comment:
This is equivalent without a bunch of extra intermediate promises. 😅 

```suggestion
  await _tracer.trace('self.sleepy', () => sleep(200))
```